Continuous notification and vibration sound in app

Device: Pixel 4a
OS: CalyxOS 2.6.1
Android: 11

Nextcloud app versions: 20210616 and 3.16.1

Recently the app continually plays a notification sound and vibration sporadically whilst the app is in the foreground. Even with all notifications disabled for the app. This even occurs having re-installed the app and the app is displaying the startup, enter server address screen. It is incredibly annoying!

Filtered logcat output is here:

```
06-20 09:31:58.914  1588 10234 I ActivityTaskManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity bnds=[855,1534][1065,1767]} from uid 10184
06-20 09:32:01.274  1588  6366 W ActivityTaskManager: Duplicate finish request for r=ActivityRecord{c82e95f u0 com.nextcloud.client/com.owncloud.android.ui.activity.SettingsActivity t3584 f}}
06-20 09:33:56.891  1588  8618 I ActivityTaskManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity bnds=[855,1534][1065,1767]} from uid 10184
06-20 09:34:01.907  1588  4703 I ActivityTaskManager: START u0 {act=android.settings.APPLICATION_DETAILS_SETTINGS dat=package:com.nextcloud.client flg=0x10008000 cmp=com.android.settings/.applications.InstalledAppDetails bnds=[132,489][948,742]} from uid 10152
06-20 09:34:03.867  1588  6366 I ActivityManager: Force stopping com.nextcloud.client appid=10250 user=0: from pid 16513
06-20 09:34:03.868  1588  6366 I ActivityManager: Killing 18035:com.nextcloud.client/u0a250 (adj 900): stop com.nextcloud.client due to from pid 16513
06-20 09:34:03.871  1588  6366 W ActivityTaskManager: Force removing ActivityRecord{c64931d u0 com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity t3584 f}}: app died, no saved state
06-20 09:34:03.893  1588  1702 W BroadcastQueue: Background execution not allowed: receiving Intent { act=android.intent.action.PACKAGE_RESTARTED dat=package:com.nextcloud.client flg=0x10 (has extras) } to com.google.android.gms/org.microg.nlp.service.PackageChangedReceiver
06-20 09:34:07.001  1588 11112 I ActivityTaskManager: START u0 {act=android.intent.action.VIEW cmp=com.nextcloud.client/com.owncloud.android.ui.activity.ManageSpaceActivity} from uid 1000
06-20 09:34:07.025  1588  1703 I ActivityManager: Start proc 18485:com.nextcloud.client/u0a250 for pre-top-activity {com.nextcloud.client/com.owncloud.android.ui.activity.ManageSpaceActivity}
06-20 09:34:07.102  1588  6366 D ConnectivityService: requestNetwork for uid/pid:10250/18485 NetworkRequest [ TRACK_DEFAULT id=2198, [ Capabilities: INTERNET&NOT_RESTRICTED&TRUSTED Uid: 10250 AdministratorUids: [] RequestorUid: 10250 RequestorPackageName: com.nextcloud.client] ]
06-20 09:34:08.967  1588  2013 D ConnectivityService: releasing NetworkRequest [ TRACK_DEFAULT id=2198, [ Capabilities: INTERNET&NOT_RESTRICTED&TRUSTED Uid: 10250 AdministratorUids: [] RequestorUid: 10250 RequestorPackageName: com.nextcloud.client] ] (release request)
06-20 09:34:16.331  1588  5861 I ActivityTaskManager: START u0 {act=android.intent.action.VIEW cmp=com.nextcloud.client/com.owncloud.android.ui.activity.ManageSpaceActivity} from uid 1000
06-20 09:34:20.664  1588 10234 I ActivityTaskManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10300000 cmp=com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity} from uid 10184
06-20 09:34:20.752 18485 18603 E AndroidRuntime: Process: com.nextcloud.client, PID: 18485
06-20 09:34:20.762  1588  5861 I ActivityTaskManager: START u0 {flg=0x10000000 cmp=com.nextcloud.client/.errorhandling.ShowErrorActivity (has extras)} from uid 10250
06-20 09:34:20.792  1588  8432 W ActivityTaskManager:   Force finishing activity com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity
06-20 09:34:20.807  1588  1703 I ActivityManager: Start proc 18608:com.nextcloud.client:crash/u0a250 for pre-top-activity {com.nextcloud.client/com.nextcloud.client.errorhandling.ShowErrorActivity}
06-20 09:34:20.835  1588  5861 I ActivityManager: Process com.nextcloud.client (pid 18485) has died: fg  TOP 
06-20 09:34:20.951  1588  1703 I ActivityManager: Start proc 18647:com.nextcloud.client/u0a250 for content provider {com.nextcloud.client/com.owncloud.android.providers.FileContentProvider}
06-20 09:34:21.271  1588  1685 W ActivityTaskManager: Activity top resumed state loss timeout for ActivityRecord{35c1d91 u0 com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity t-1 f}}
06-20 09:34:21.380  1588  2292 D ConnectivityService: requestNetwork for uid/pid:10250/18647 NetworkRequest [ TRACK_DEFAULT id=2199, [ Capabilities: INTERNET&NOT_RESTRICTED&TRUSTED Uid: 10250 AdministratorUids: [] RequestorUid: 10250 RequestorPackageName: com.nextcloud.client] ]
06-20 09:34:21.949  1588  2013 D ConnectivityService: releasing NetworkRequest [ TRACK_DEFAULT id=2199, [ Capabilities: INTERNET&NOT_RESTRICTED&TRUSTED Uid: 10250 AdministratorUids: [] RequestorUid: 10250 RequestorPackageName: com.nextcloud.client] ] (release request)
06-20 09:34:36.978  1588  1685 I ActivityManager: Killing 18647:com.nextcloud.client/u0a250 (adj 700): remove task
06-20 09:34:36.979  1588  1685 I ActivityManager: Killing 18608:com.nextcloud.client:crash/u0a250 (adj 905): remove task
06-20 09:34:37.028  1588  2292 I WindowManager: WIN DEATH: Window{9b46d6c u0 com.nextcloud.client/com.nextcloud.client.errorhandling.ShowErrorActivity}
06-20 09:34:37.879  1588 10234 I ActivityTaskManager: START u0 {act=android.settings.APPLICATION_DETAILS_SETTINGS dat=package:com.nextcloud.client flg=0x10108000 cmp=com.android.settings/.applications.InstalledAppDetails} from uid 10152
06-20 09:34:38.814  1588 16612 I ActivityTaskManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10000000 pkg=com.nextcloud.client cmp=com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity} from uid 1000
06-20 09:34:38.843  1588  1703 I ActivityManager: Start proc 18822:com.nextcloud.client/u0a250 for pre-top-activity {com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity}
06-20 09:34:48.301  1588  2292 I ActivityTaskManager: START u0 {flg=0x4000000 cmp=com.nextcloud.client/com.owncloud.android.ui.activity.UploadListActivity} from uid 10250
06-20 09:34:51.706  1588 11108 I ActivityTaskManager: START u0 {cmp=com.nextcloud.client/com.owncloud.android.ui.activity.NotificationsActivity} from uid 10250
06-20 09:35:02.237  1588 16612 I ActivityTaskManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity bnds=[855,1534][1065,1767]} from uid 10184
06-20 09:35:06.091  1588 16612 I ActivityTaskManager: START u0 {act=android.settings.APPLICATION_DETAILS_SETTINGS dat=package:com.nextcloud.client flg=0x10008000 cmp=com.android.settings/.applications.InstalledAppDetails bnds=[132,489][948,742]} from uid 10152
06-20 09:35:07.144  1588 11051 E UserRestrictionsUtils: Unknown restriction queried by uid 1000 (com.android.providers.settings et al): com.nextcloud.client
06-20 09:35:07.147  1588 11051 I ActivityTaskManager: START u0 {act=android.intent.action.UNINSTALL_PACKAGE dat=package:com.nextcloud.client cmp=com.android.packageinstaller/.UninstallerActivity (has extras)} from uid 1000
06-20 09:35:07.964  1588  1702 I ActivityManager: Force stopping com.nextcloud.client appid=10250 user=0: deletePackageX
06-20 09:35:07.964  1588  1702 I ActivityManager: Killing 18822:com.nextcloud.client/u0a250 (adj 900): stop com.nextcloud.client due to deletePackageX
06-20 09:35:07.966  1588  1702 W ActivityTaskManager: Force removing ActivityRecord{aa1b4cf u0 com.nextcloud.client/com.owncloud.android.ui.activity.NotificationsActivity t3587 f}}: app died, no saved state
06-20 09:35:07.966  1588  1702 W ActivityTaskManager: Force removing ActivityRecord{52b6037 u0 com.nextcloud.client/com.owncloud.android.ui.activity.UploadListActivity t3587 f}}: app died, no saved state
06-20 09:35:07.966  1588  1702 W ActivityTaskManager: Force removing ActivityRecord{4bc69b5 u0 com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity t3587 f}}: app died, no saved state
06-20 09:35:07.967  1588  1702 I WindowManager:   Force finishing activity ActivityRecord{aa1b4cf u0 com.nextcloud.client/com.owncloud.android.ui.activity.NotificationsActivity t3587 f} isExiting}
06-20 09:35:07.968  1588  1702 W ActivityTaskManager: Duplicate finish request for r=ActivityRecord{aa1b4cf u0 com.nextcloud.client/com.owncloud.android.ui.activity.NotificationsActivity t3587 f} isExiting}
06-20 09:35:07.968  1588  1702 I WindowManager:   Force finishing activity ActivityRecord{52b6037 u0 com.nextcloud.client/com.owncloud.android.ui.activity.UploadListActivity t3587 f} isExiting}
06-20 09:35:07.968  1588  1702 W ActivityTaskManager: Duplicate finish request for r=ActivityRecord{52b6037 u0 com.nextcloud.client/com.owncloud.android.ui.activity.UploadListActivity t3587 f} isExiting}
06-20 09:35:07.968  1588  1702 I WindowManager:   Force finishing activity ActivityRecord{4bc69b5 u0 com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity t3587 f} isExiting}
06-20 09:35:07.968  1588  1702 W ActivityTaskManager: Duplicate finish request for r=ActivityRecord{4bc69b5 u0 com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity t3587 f} isExiting}
06-20 09:35:08.030  1588 11051 I WindowManager: WIN DEATH: Window{e999c43 u0 com.nextcloud.client/com.owncloud.android.ui.activity.NotificationsActivity}
06-20 09:35:08.033  1588  2292 I WindowManager: WIN DEATH: Window{7db2fb u0 com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity}
06-20 09:35:08.035  1588 16612 I WindowManager: WIN DEATH: Window{c718815 u0 com.nextcloud.client/com.owncloud.android.ui.activity.UploadListActivity}
06-20 09:35:08.294  1588  1837 I ActivityManager: Force stopping com.nextcloud.client appid=10250 user=0: pkg removed
06-20 09:35:08.294  1588  1837 I WindowManager:   Force finishing activity ActivityRecord{aa1b4cf u0 com.nextcloud.client/com.owncloud.android.ui.activity.NotificationsActivity t3587 f} isExiting}
06-20 09:35:08.294  1588  1837 W ActivityTaskManager: Duplicate finish request for r=ActivityRecord{aa1b4cf u0 com.nextcloud.client/com.owncloud.android.ui.activity.NotificationsActivity t3587 f} isExiting}
06-20 09:35:08.294  1588  1837 I WindowManager:   Force finishing activity ActivityRecord{52b6037 u0 com.nextcloud.client/com.owncloud.android.ui.activity.UploadListActivity t3587 f} isExiting}
06-20 09:35:08.294  1588  1837 W ActivityTaskManager: Duplicate finish request for r=ActivityRecord{52b6037 u0 com.nextcloud.client/com.owncloud.android.ui.activity.UploadListActivity t3587 f} isExiting}
06-20 09:35:08.294  1588  1837 I WindowManager:   Force finishing activity ActivityRecord{4bc69b5 u0 com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity t3587 f} isExiting}
06-20 09:35:08.294  1588  1837 W ActivityTaskManager: Duplicate finish request for r=ActivityRecord{4bc69b5 u0 com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity t3587 f} isExiting}
06-20 09:35:08.523  1588  1996 V StorageManagerService: Package com.nextcloud.client does not have legacy storage
06-20 09:35:08.528  1588  1682 E PackageManager: failed to find package com.nextcloud.client
06-20 09:35:08.528  1588  1682 E OverlayManager: Failed to change enabled overlays for com.nextcloud.client user 0
06-20 09:35:08.538  1588  1702 W BroadcastQueue: Background execution not allowed: receiving Intent { act=android.intent.action.PACKAGE_REMOVED dat=package:com.nextcloud.client flg=0x4000010 (has extras) } to com.android.musicfx/.Compatibility$Receiver
06-20 09:35:08.538  1588  1702 W BroadcastQueue: Background execution not allowed: receiving Intent { act=android.intent.action.PACKAGE_REMOVED dat=package:com.nextcloud.client flg=0x4000010 (has extras) } to com.google.android.gms/org.microg.gms.gcm.UnregisterReceiver
06-20 09:35:08.538  1588  1837 I ActivityManager: Force stopping com.nextcloud.client appid=10250 user=0: pkg removed
06-20 09:35:08.552  1588  1702 W BroadcastQueue: Background execution not allowed: receiving Intent { act=android.intent.action.PACKAGE_REMOVED dat=package:com.nextcloud.client flg=0x4000010 (has extras) } to com.google.android.gms/org.microg.nlp.service.PackageChangedReceiver
06-20 09:35:08.581  1588  1975 I RollbackManager: broadcast=ACTION_PACKAGE_FULLY_REMOVED pkg=com.nextcloud.client
06-20 09:35:08.614  1588  2325 W BroadcastQueue: Background execution not allowed: receiving Intent { act=android.intent.action.PACKAGE_REMOVED dat=package:com.nextcloud.client flg=0x4000010 (has extras) } to com.aurora.store/.data.receiver.PackageManagerReceiver
06-20 09:35:08.616  1588  1702 W BroadcastQueue: Background execution not allowed: receiving Intent { act=android.intent.action.PACKAGE_REMOVED dat=package:com.nextcloud.client flg=0x4000010 (has extras) } to com.google.android.apps.photos/com.google.android.libraries.social.peoplekit.thirdparty.viewcontrollers.ThirdPartyReceiver
06-20 09:35:08.616  1588  1702 W BroadcastQueue: Background execution not allowed: receiving Intent { act=android.intent.action.PACKAGE_REMOVED dat=package:com.nextcloud.client flg=0x4000010 (has extras) } to com.microsoft.launcher/com.microsoft.mmx.continuity.receiver.InstallUninstallReceiver
06-20 09:35:08.750  1588  1588 I ConditionProviders: Disallowing condition provider com.nextcloud.client
06-20 09:35:40.033  1588  1837 V BackupManagerService: [UserID:0] restoreAtInstall pkg=com.nextcloud.client token=71 restoreSet=0
06-20 09:35:40.066  1588  1702 W BroadcastQueue: Background execution not allowed: receiving Intent { act=android.intent.action.PACKAGE_ADDED dat=package:com.nextcloud.client flg=0x4000010 (has extras) } to com.android.packageinstaller/.PackageInstalledReceiver
06-20 09:35:40.108  1588  2023 W BroadcastQueue: Background execution not allowed: receiving Intent { act=android.intent.action.PACKAGE_ADDED dat=package:com.nextcloud.client flg=0x4000010 (has extras) } to com.aurora.store/.data.receiver.PackageManagerReceiver
06-20 09:35:40.109  1588  1702 W BroadcastQueue: Background execution not allowed: receiving Intent { act=android.intent.action.PACKAGE_ADDED dat=package:com.nextcloud.client flg=0x4000010 (has extras) } to com.microsoft.launcher/com.microsoft.mmx.continuity.receiver.InstallUninstallReceiver
06-20 09:35:44.436  1588 16927 I ActivityTaskManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity bnds=[15,835][225,1068]} from uid 10184
06-20 09:35:44.467  1588  1703 I ActivityManager: Start proc 19204:com.nextcloud.client/u0a282 for pre-top-activity {com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity}
06-20 09:35:44.655  1588 16612 D ConnectivityService: requestNetwork for uid/pid:10282/19204 NetworkRequest [ TRACK_DEFAULT id=2200, [ Capabilities: INTERNET&NOT_RESTRICTED&TRUSTED Uid: 10282 AdministratorUids: [] RequestorUid: 10282 RequestorPackageName: com.nextcloud.client] ]
06-20 09:35:44.808  1588 16612 I ActivityTaskManager: START u0 {flg=0x10800004 cmp=com.nextcloud.client/com.owncloud.android.authentication.AuthenticatorActivity clip={text/plain {...}} (has extras)} from uid 10282
06-20 09:35:44.817  1588  2292 I ActivityTaskManager: START u0 {flg=0x10800004 cmp=com.nextcloud.client/com.owncloud.android.authentication.AuthenticatorActivity clip={text/plain {...}} (has extras)} from uid 10282
06-20 09:35:45.152  1588 11051 I ActivityTaskManager: START u0 {cmp=com.nextcloud.client/.onboarding.FirstRunActivity} from uid 10282
06-20 09:35:45.339  1588  2013 D ConnectivityService: releasing NetworkRequest [ TRACK_DEFAULT id=2200, [ Capabilities: INTERNET&NOT_RESTRICTED&TRUSTED Uid: 10282 AdministratorUids: [] RequestorUid: 10282 RequestorPackageName: com.nextcloud.client] ] (release request)
06-20 09:35:55.907  1588 10234 I ActivityTaskManager: START u0 {flg=0x10800004 cmp=com.nextcloud.client/com.owncloud.android.authentication.AuthenticatorActivity clip={text/plain {...}} (has extras)} from uid 10282
06-20 09:36:02.017  1588  8271 I ActivityTaskManager: START u0 {act=android.settings.APPLICATION_DETAILS_SETTINGS dat=package:com.nextcloud.client flg=0x10008000 cmp=com.android.settings/.applications.InstalledAppDetails bnds=[132,489][948,742]} from uid 10152
06-20 09:54:09.066  1588 11108 I ActivityTaskManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity bnds=[15,835][225,1068]} from uid 10184
06-20 09:54:21.566  1588  2325 I ActivityTaskManager: START u0 {act=android.settings.APPLICATION_DETAILS_SETTINGS dat=package:com.nextcloud.client flg=0x10008000 cmp=com.android.settings/.applications.InstalledAppDetails bnds=[197,634][373,832]} from uid 10184
06-20 09:54:28.171  1588  3265 I ActivityTaskManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10000000 pkg=com.nextcloud.client cmp=com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity} from uid 1000
06-20 09:54:51.082  1588  2325 I ActivityTaskManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity bnds=[15,835][225,1068]} from uid 10184
06-20 09:55:00.144  1588 11108 I ActivityTaskManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity bnds=[15,835][225,1068]} from uid 10184
06-20 09:55:07.100  1588  1626 I ActivityTaskManager: START u0 {act=android.settings.APPLICATION_DETAILS_SETTINGS dat=package:com.nextcloud.client flg=0x10008000 cmp=com.android.settings/.applications.InstalledAppDetails bnds=[197,634][373,832]} from uid 10184
06-20 09:55:21.823  1588  8271 I ActivityTaskManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity bnds=[15,835][225,1068]} from uid 10184
06-20 09:55:33.648  1588  8271 I ActivityTaskManager: START u0 {act=android.settings.APPLICATION_DETAILS_SETTINGS dat=package:com.nextcloud.client flg=0x10008000 cmp=com.android.settings/.applications.InstalledAppDetails bnds=[197,634][373,832]} from uid 10184
06-20 09:55:47.816  1588 16927 I ActivityTaskManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity bnds=[15,835][225,1068]} from uid 10184
06-20 09:56:14.220  1588 12754 I ActivityTaskManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity bnds=[15,835][225,1068]} from uid 10184
06-20 10:01:55.198  1588  1588 I NotificationService: Cannot find enqueued record for key: 0|org.fdroid.fdroid|1|https://f-droid.org/repo/com.nextcloud.android.beta_20210616.apk|10123
06-20 10:01:58.668  1588  1588 D NotificationService: 0|org.fdroid.fdroid|1|https://f-droid.org/repo/com.nextcloud.android.beta_20210616.apk|10123: granting content://settings/system/notification_sound
06-20 10:01:58.669  1588  1588 D NotificationService: 0|org.fdroid.fdroid|1|https://f-droid.org/repo/com.nextcloud.android.beta_20210616.apk|10123: granting content://settings/system/notification_sound
06-20 10:02:01.643  1588  1837 V BackupManagerService: [UserID:0] restoreAtInstall pkg=com.nextcloud.android.beta token=72 restoreSet=0
06-20 10:02:01.673  1588  1702 W BroadcastQueue: Background execution not allowed: receiving Intent { act=android.intent.action.PACKAGE_ADDED dat=package:com.nextcloud.android.beta flg=0x4000010 (has extras) } to com.android.packageinstaller/.PackageInstalledReceiver
06-20 10:02:01.716  1588  5904 W BroadcastQueue: Background execution not allowed: receiving Intent { act=android.intent.action.PACKAGE_ADDED dat=package:com.nextcloud.android.beta flg=0x4000010 (has extras) } to com.aurora.store/.data.receiver.PackageManagerReceiver
06-20 10:02:01.717  1588  1702 W BroadcastQueue: Background execution not allowed: receiving Intent { act=android.intent.action.PACKAGE_ADDED dat=package:com.nextcloud.android.beta flg=0x4000010 (has extras) } to com.microsoft.launcher/com.microsoft.mmx.continuity.receiver.InstallUninstallReceiver
06-20 10:02:02.033  1588  1588 D NotificationService: 0|org.fdroid.fdroid|2|https://f-droid.org/repo/com.nextcloud.android.beta_20210616.apk|10123: granting content://settings/system/notification_sound
06-20 10:02:02.033  1588  1588 D NotificationService: 0|org.fdroid.fdroid|2|https://f-droid.org/repo/com.nextcloud.android.beta_20210616.apk|10123: granting content://settings/system/notification_sound
06-20 10:02:03.513  1588 16927 I ActivityTaskManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10000000 pkg=com.nextcloud.android.beta cmp=com.nextcloud.android.beta/com.owncloud.android.ui.activity.FileDisplayActivity} from uid 10123
06-20 10:02:03.554  1588  1703 I ActivityManager: Start proc 20514:com.nextcloud.android.beta/u0a283 for pre-top-activity {com.nextcloud.android.beta/com.owncloud.android.ui.activity.FileDisplayActivity}
06-20 10:02:03.769  1588  8271 D ConnectivityService: requestNetwork for uid/pid:10283/20514 NetworkRequest [ TRACK_DEFAULT id=2209, [ Capabilities: INTERNET&NOT_RESTRICTED&TRUSTED Uid: 10283 AdministratorUids: [] RequestorUid: 10283 RequestorPackageName: com.nextcloud.android.beta] ]
06-20 10:02:03.927  1588 16927 I ActivityTaskManager: START u0 {flg=0x10800004 cmp=com.nextcloud.android.beta/com.owncloud.android.authentication.AuthenticatorActivity clip={text/plain {...}} (has extras)} from uid 10283
06-20 10:02:03.939  1588  3403 I ActivityTaskManager: START u0 {flg=0x10800004 cmp=com.nextcloud.android.beta/com.owncloud.android.authentication.AuthenticatorActivity clip={text/plain {...}} (has extras)} from uid 10283
06-20 10:02:04.273  1588  8271 I ActivityTaskManager: START u0 {cmp=com.nextcloud.android.beta/com.nextcloud.client.onboarding.FirstRunActivity} from uid 10283
06-20 10:02:04.439  1588  2013 D ConnectivityService: releasing NetworkRequest [ TRACK_DEFAULT id=2209, [ Capabilities: INTERNET&NOT_RESTRICTED&TRUSTED Uid: 10283 AdministratorUids: [] RequestorUid: 10283 RequestorPackageName: com.nextcloud.android.beta] ] (release request)
06-20 10:24:12.211 20514 20539 I WM-WorkerWrapper: Worker result SUCCESS for Work [ id=f8d6b767-ec44-430e-b3c0-cc4332f2a955, tags={ timestamp:1624179723726, com.nextcloud.client.jobs.OfflineSyncWork, *, name:periodic_offline_sync } ]
06-20 10:24:12.249 20514 20539 I WM-WorkerWrapper: Worker result SUCCESS for Work [ id=398dea5a-a431-4880-afcf-ef1083ccc8fb, tags={ name:periodic_files_sync, timestamp:1624179723714, com.nextcloud.client.jobs.FilesSyncWork, * } ]
06-20 10:24:12.269 20514 20542 I WM-WorkerWrapper: Worker result SUCCESS for Work [ id=b4930121-a163-4ca4-a762-e07ac8475648, tags={ com.nextcloud.client.jobs.MediaFoldersDetectionWork, timestamp:1624179723745, *, name:periodic_media_folder_detection } ]
06-20 10:34:59.582 19204 19227 I WM-WorkerWrapper: Worker result SUCCESS for Work [ id=8a435667-a8da-489e-9e9f-770df2c4cf86, tags={ timestamp:1624178144627, com.nextcloud.client.jobs.OfflineSyncWork, *, name:periodic_offline_sync } ]
06-20 10:34:59.613 19204 19233 I WM-WorkerWrapper: Worker result SUCCESS for Work [ id=3867f8c1-c29e-4688-b9c3-0ee886239a81, tags={ name:periodic_files_sync, timestamp:1624178144619, com.nextcloud.client.jobs.FilesSyncWork, * } ]
06-20 10:34:59.635 19204 19227 I WM-WorkerWrapper: Worker result SUCCESS for Work [ id=b63d2e20-64ca-45b7-8c27-38d478fa928b, tags={ com.nextcloud.client.jobs.MediaFoldersDetectionWork, *, timestamp:1624178144641, name:periodic_media_folder_detection } ]
06-20 10:38:08.954  1588  5904 I ActivityTaskManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=com.nextcloud.android.beta/com.owncloud.android.ui.activity.FileDisplayActivity bnds=[225,835][435,1068]} from uid 10184
06-20 10:38:10.052  3314  3314 I AndroidIME: AbstractIme.onActivate():88 LatinIme.onActivate() : EditorInfo = Package = com.nextcloud.android.beta : Type = Text : Learning = Disable : Suggestion = Hide : AutoCorrection = Disable : Microphone = Show : NoPersonalizedLearning = Disable : AutoStartVoiceInput = Disable : AutoStartEmojiInput = Disable : SimulateDeviceLocked = False : imeActionName = NEXT : imeOptions = 5 : privateImeOptions = null, IncognitoMode = false, DeviceLocked = false
06-20 10:38:10.059  3314  3314 I ConversationToQueryExtension: ConversationToQueryExtension.isEnabledForHostApp():222 Conv2Query not enabled due to current app [com.nextcloud.android.beta] not in allowlist
06-20 10:38:18.014 21340 21456 D com.nextcloud.android.sso.api.NetworkRequest: [connectApiWithBackoff] connectApiWithBackoff() called from Thread: [Thread-2]
06-20 10:38:18.022 21340 21340 D com.nextcloud.android.sso.api.NetworkRequest: [connectApiWithBackoff] trying to connect..
06-20 10:38:18.022 21340 21340 D com.nextcloud.android.sso.api.AidlNetworkRequest: [connect] Binding to AccountManagerService for type [nextcloud]
06-20 10:38:18.022 21340 21340 D com.nextcloud.android.sso.api.NetworkRequest: [connect] connect() called [main] Account-Type: [nextcloud]
06-20 10:38:18.022 21340 21340 D com.nextcloud.android.sso.api.AidlNetworkRequest: [connect] Component name is: [com.nextcloud.client]
06-20 10:38:18.026 21340 21340 D com.nextcloud.android.sso.api.AidlNetworkRequest: [connect] Bound to AccountManagerService successfully
06-20 10:38:18.026 21340 21340 D com.nextcloud.android.sso.api.AidlNetworkRequest: [onServiceConnected] called from Thread: [main] with IBinder [ComponentInfo{com.nextcloud.client/com.owncloud.android.services.AccountManagerService}]: android.os.BinderProxy@378d54e
06-20 10:38:18.027 21340 21340 I ApiProvider: SSO API connected for com.nextcloud.android.sso.model.SingleSignOnAccount@3c4c66f
06-20 10:38:18.061 21340 21447 D com.nextcloud.android.sso.api.NextcloudRetrofitServiceMethod: NextcloudRetrofitServiceMethod() called with: apiEndpoint = [/index.php/apps/notes/api/v1/], method = [public abstract io.reactivex.Observable it.niedermann.owncloud.notes.persistence.sync.NotesAPI_1_0.getNotes(long,java.lang.String)]
06-20 10:38:18.062 21340 21447 D com.nextcloud.android.sso.api.NextcloudRetrofitServiceMethod: invoke call to api using observable com.nextcloud.android.sso.api.ParsedResponse<java.util.List<it.niedermann.owncloud.notes.persistence.entity.Note>>
06-20 10:38:18.075 21340 21468 D com.nextcloud.android.sso.api.AidlNetworkRequest: copy data from service finished
06-20 10:38:18.078 19204 19548 E InputStreamBinder: 	at com.nextcloud.android.sso.InputStreamBinder.processRequestV2(InputStreamBinder.java:377)
06-20 10:38:18.078 19204 19548 E InputStreamBinder: 	at com.nextcloud.android.sso.InputStreamBinder.performNextcloudRequestAndBodyStreamV2(InputStreamBinder.java:125)
06-20 10:38:18.078 19204 19548 E InputStreamBinder: 	at com.nextcloud.android.sso.InputStreamBinder.performNextcloudRequestV2(InputStreamBinder.java:108)
06-20 10:38:18.078 19204 19548 E InputStreamBinder: 	at com.nextcloud.android.sso.aidl.IInputStreamService$Stub.onTransact(IInputStreamService.java:158)
06-20 10:38:18.095 21340 21447 E NotesRepository: com.nextcloud.android.sso.exceptions.NextcloudFilesAppAccountNotFoundException: Account not found in nextcloud files app
06-20 10:38:18.095 21340 21447 E NotesRepository: java.lang.RuntimeException: com.nextcloud.android.sso.exceptions.NextcloudFilesAppAccountNotFoundException: Account not found in nextcloud files app
06-20 10:38:18.095 21340 21447 E NotesRepository: Caused by: com.nextcloud.android.sso.exceptions.NextcloudFilesAppAccountNotFoundException: Account not found in nextcloud files app
06-20 10:38:18.095 21340 21447 E NotesRepository: 	at com.nextcloud.android.sso.api.AidlNetworkRequest.performNetworkRequestV2(AidlNetworkRequest.java:172)
06-20 10:38:18.095 21340 21447 E NotesRepository: 	at com.nextcloud.android.sso.api.NextcloudAPI.performNetworkRequestV2(NextcloudAPI.java:171)
06-20 10:38:18.095 21340 21447 E NotesRepository: 	at com.nextcloud.android.sso.api.NextcloudAPI.lambda$performRequestObservableV2$1$NextcloudAPI(NextcloudAPI.java:110)
06-20 10:38:18.095 21340 21447 E NotesRepository: 	at com.nextcloud.android.sso.api.-$$Lambda$NextcloudAPI$8WS9mapJbOYsPgbhBi25vBNYZqI.subscribe(Unknown Source:6)
06-20 10:38:18.311  1588  1959 I ActivityTaskManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity bnds=[15,835][225,1068]} from uid 10184
06-20 10:38:18.463  3314  3314 I AndroidIME: AbstractIme.onActivate():88 LatinIme.onActivate() : EditorInfo = Package = com.nextcloud.client : Type = Text : Learning = Disable : Suggestion = Hide : AutoCorrection = Disable : Microphone = Show : NoPersonalizedLearning = Disable : AutoStartVoiceInput = Disable : AutoStartEmojiInput = Disable : SimulateDeviceLocked = False : imeActionName = NEXT : imeOptions = 5 : privateImeOptions = null, IncognitoMode = false, DeviceLocked = false
06-20 10:38:18.467  3314  3314 I ConversationToQueryExtension: ConversationToQueryExtension.isEnabledForHostApp():222 Conv2Query not enabled due to current app [com.nextcloud.client] not in allowlist
06-20 10:38:19.645 19204 21469 E com.nextcloud.android.sso.aidl.ParcelFileDescriptorUtil.TransferThread: writing failed: write failed: EPIPE (Broken pipe)
```

--- a/app/src/main/java/com/owncloud/android/MainApp.java
+++ b/app/src/main/java/com/owncloud/android/MainApp.java
@@ -30,6 +30,7 @@ import android.app.NotificationManager;
 import android.content.ContentResolver;
 import android.content.Context;
 import android.content.Intent;
+import android.content.pm.ActivityInfo;
 import android.content.pm.PackageInfo;
 import android.content.pm.PackageManager;
 import android.os.Build;
@@ -57,6 +58,7 @@ import com.nextcloud.client.onboarding.OnboardingService;
 import com.nextcloud.client.preferences.AppPreferences;
 import com.nextcloud.client.preferences.AppPreferencesImpl;
 import com.nextcloud.client.preferences.DarkMode;
+import com.owncloud.android.authentication.AuthenticatorActivity;
 import com.owncloud.android.authentication.PassCodeManager;
 import com.owncloud.android.datamodel.ArbitraryDataProvider;
 import com.owncloud.android.datamodel.MediaFolder;
@@ -370,12 +372,33 @@ public class MainApp extends MultiDexApplication implements HasAndroidInjector {
         });
     }
 
+    @SuppressWarnings("unchecked")
     private void initSecurityKeyManager() {
         SecurityKeyManager securityKeyManager = SecurityKeyManager.getInstance();
-        SecurityKeyManagerConfig config = new SecurityKeyManagerConfig.Builder()
-            .setEnableDebugLogging(BuildConfig.DEBUG)
-            .build();
-        securityKeyManager.init(this, config);
+        final SecurityKeyManagerConfig.Builder configBuilder = new SecurityKeyManagerConfig.Builder()
+            .setEnableDebugLogging(BuildConfig.DEBUG);
+
+        try {
+            // exclude all activities except AuthenticatorActivity
+            final PackageManager pm = this.getPackageManager();
+            final PackageInfo info = pm.getPackageInfo(this.getPackageName(), PackageManager.GET_ACTIVITIES);
+            final ActivityInfo[] activities = info.activities;
+            for (ActivityInfo activityInfo : activities) {
+                try {
+                    final Class<? extends Activity> aClass = (Class<? extends Activity>) Class.forName(activityInfo.name);
+                    if (aClass != AuthenticatorActivity.class) {
+                        configBuilder.addExcludedActivityClass(aClass);
+                    }
+                } catch (ClassNotFoundException | ClassCastException e) {
+                    Log_OC.e(TAG, "Couldn't disable activity for security key listener", e);
+                }
+            }
+        } catch (PackageManager.NameNotFoundException e) {
+            Log_OC.e(TAG, "Couldn't disable activities for security key listener", e);
+        }
+
+
+        securityKeyManager.init(this, configBuilder.build());
     }
 
     public static void initContactsBackup(UserAccountManager accountManager, BackgroundJobManager backgroundJobManager) {

